cli/src/commands/parse.rs

```diff
@@ -275,6 +285,23 @@ where
     None
 }
 
+/// Strip root prefix from lockfile paths
+fn strip_root_path(path: PathBuf, project_root: &Option<PathBuf>) -> PathBuf {
```

Comment:
We should probably do canonicalization. Especially in the case where the project root exists already, we don't want to be relative to the current directory.

Reply:
I think I covered the issue you brought up. Previously running analyze in a subdirectory was returning path relative to that directory vs the project.